source/agora/common/Config.d

```diff
@@ -183,6 +183,9 @@ public struct ValidatorConfig
 
     // Registry address
     public string registry_address;
+
+    // If the enrollments will be renewed or not at the end of the cycle
+    public bool recurring_enrollment;
```

Comment:
This needs to be added in the documented config.

Reply:
Where is that exactly? :) I think it should be added to the parsing from file routines as well.

Reply:
This one: https://github.com/bpfkorea/agora/blob/v0.x.x/doc/config.example.yaml